mock.js

```diff
@@ -230,7 +233,10 @@ module.exports = {
 
   ...Reanimated,
 
-  default: Reanimated,
+  default: {
+    ...Reanimated,
+    ...ReanimatedV2,
+  },
```

Comment:
Late to the party but I'd like to ask regardless...

I see you are introducing `default` export here.

With given setup instructions, mock are not initialised properly
```
jest.mock('react-native-reanimated', () => require('react-native-reanimated/mock'));
```

Not unless I use `default` export.

```
jest.mock('react-native-reanimated', () => require('react-native-reanimated/mock').default);
```

I believe your approach with `default` export is correct, however usage docs should be updated.

According to implementation (index.js) named exports of:
```
...Reanimated
```
should be dropped, since only these name exports are specified
```
export {
  EasingNode,
  Transitioning,
  Transition,
  createTransitioningComponent,
  // classes
  AnimatedClock as Clock,
  AnimatedValue as Value,
  AnimatedNode as Node,
  // animations
  decayWrapper as decay,
  timingWrapper as timing,
  springWrapper as spring,
  SpringUtils,
  // hooks
  useValue,
};
```

I am correct or is there something I am missing?

Reply:
@Andreyco when exporting with `__esModule: true` Jest will know how to deal with default imports correctly. No change is needed.

About the V1 exports, we export Reanimated V1 APIs for backward compatibility, right?

Reply:
I cannot confirm your statement. Just did a test run, even with `__esModule: true`, Jest is... lost.

Seems proper mock initialisation is, indeed, following...
```js
jest.mock('react-native-reanimated', () => require('react-native-reanimated/mock').default);
```

@mgambati could I ask you to double check with your Jest setup?
My setup is:
Jest: 26.6.3
RNR: current master (https://github.com/software-mansion/react-native-reanimated.git#ba4d65e25c08493e2ad0b53d10344a2dd72bdafe)